### PR TITLE
added the google font Ubuntu

### PR DESCRIPTION
**Pull request by ayush015**

### CODE REVIEW

**Comment by ayush015** [comment]
Screenshot 2020-10-13 at 1 24 40 AM (2)
Screenshot 2020-10-13 at 1 24 54 AM (2)

screenshots for the newly added font

**Comment by iamdarshshah** [comment]
@ayush015 This looks good! but I would insist you to go through this article - [20 Best Google Web Fonts](https://www.awwwards.com/20-best-web-fonts-from-google-web-fonts-and-font-face.html) & add some of it to this (at max. 5).

Also, this PR has some conflicts with the master branch. Make sure you resolve it. Thanks

**Comment by ayush015** [comment]
Okay I'll make the changes Thanks for the review

**Comment by ayush015** [comment]
@iamdarshshah  in this given [20 Best Google Web Fonts ](https://www.awwwards.com/20-best-web-fonts-from-google-web-fonts-and-font-face.html) list to have any recommendations that i should add ?

**Comment by iamdarshshah** [comment]
Hey, @ayush015 this PR has some conflicts. Please resolve it. Also, add the fonts which you have added in `gatsby-config.js` file to the `theme.js` file within the `font-families` array.

**Comment by ayush015** [comment]
> Hey, @ayush015 this PR has some conflicts. Please resolve it. Also, add the fonts which you have added in `gatsby-config.js` file to the `theme.js` file within the `font-families` array.

Yeah i did that . I have a question I can just a one font at a time right?

**Comment by ayush015** [comment]
There is a merge conflict for src/contributors/raneyash.json. I'm not under standing how to resolve it

**Comment by iamdarshshah** [comment]
> There is a merge conflict for src/contributors/raneyash.json. I'm not under standing how to resolve it

Just delete that file.

> Yeah i did that . I have a question I can just a one font at a time right?

Nope, that's not the case. You can always add multiple font-families, Please refer to this - [here](https://material-ui.com/customization/typography/#font-family)

**Comment by ayush015** [comment]
> > There is a merge conflict for src/contributors/raneyash.json. I'm not under standing how to resolve it
> 
> Just delete that file.
> 
> > Yeah i did that . I have a question I can just a one font at a time right?
> 
> Nope, that's not the case. You can always add multiple font-families, Please refer to this - [here](https://material-ui.com/customization/typography/#font-family)

Yeah got that .
Here is what i was thinking . we can do something like after every refresh font family would change automatically ?

**Comment by ayush015** [comment]
And Done all the changes #107

File: src/contributors/gilgold.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Gil Goldstein",
+    "desc": "Software Engineer at Indeed | Creator of Tabox for Chrome",
+    "github": "https://github.com/gilgold",
+    "twitter": "https://twitter.com/gilgoldstein",
+    "linkedin": "https://www.linkedin.com/in/gilgoldstein"
+}
```
Comment (iamdarshshah) [inline src/contributors/gilgold.json:7]: Delete this file. And don't Update unwanted files. This PR should focus on adding fonts.

**Review by iamdarshshah — CHANGES_REQUESTED** [review]

**Comment by iamdarshshah** [comment]
@ayush015 Please resolve the conflicts.

File: src/contributors/gilgold.json (hunk 1 above)
Comment (ayush015) [reply src/contributors/gilgold.json]: yeah sorry for that didn't see that . Made the changes

**Review by iamdarshshah — APPROVED** [review]
LGTM! Thanks @ayush015

**Comment by iamdarshshah** [comment]
Fixes #11 🎉

**Pull request merged**